docs/rule/require-form-method-post.md

```diff
@@ -0,0 +1,30 @@
+# require-form-method-post
+
+This rule requires all `<form>` elements to have `method` attribute with `POST` value.
+
+By default `form` elements without `method` attribute are submitted as `GET` requests.
+In usual applications `submit` event listeners are attached to `form` elements and `event.preventDefault()` is called to avoid form submission.
+
+However in case of failure to prevent default action, form submission as `GET` request can leak sensitive end-user information.
```

Comment:
This should mention that there are acceptable use cases for `GET` as well: https://www.w3schools.com/tags/att_form_method.asp